source/client/options_impl.cc

```diff
@@ -78,6 +80,24 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
       "default output format is 'v4'.",
       false, "v4", &address_families_allowed, cmd);
 
+  std::vector<std::string> request_methods = {"GET",    "HEAD",    "POST",    "PUT",
```

Comment:
I'm almost wondering if we should have some way to automatically generated `tclap` (or other) human friendly flags from proto. We're doing a lot of this! :)